src/DynamoCoreWpf/Views/Core/DynamoView.xaml.cs

```diff
@@ -851,7 +851,7 @@ private void LoadSamplesMenu()
                 {
                     var showInFolder = new MenuItem
                     {
-                        Header = "Show In Folder",
+                        Header = Wpf.Properties.Resources.DynamoViewHelpMenuShowInFolder,
```

Comment:
Why can't I find this in RESX file? Is it an existing one? The same for the following two.